apps/emqx_dashboard/include/emqx_dashboard.hrl

```diff
@@ -14,7 +14,18 @@
 %% limitations under the License.
 %%--------------------------------------------------------------------
 
--record(mqtt_admin, {username, password, tags, role = undefined}).
+-record(mqtt_admin, {
+    username             :: binary(),
+    password             :: binary(),
+    tags                 :: list() | binary(),
+    role = undefined     :: atom()
+    }).
+
+-record(mqtt_admin_jwt, {
```

Comment:
It should have better record name, `mqtt_admin` too. They have nothing to do with mqtt. You can improve this in the next version.